demo/filter.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8" />
     <title>Tilegarden</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
I'm sure no one is excited at the prospect, but these demo pages are getting more comprehensive and less DRY.  It could make making changes to shared parts of them (navigation, dependencies, etc) challenging or error prone.  Since it's likely required a build step to template these, I'm not suggesting we do it in this PR, but we may either want a future issue or at least keep an eye on how frequently they get edited and make a call down the road.

Reply:
Yeah, definitely a lot of repetition happening.  I think making an issue makes sense--having demos of the range of capabilities of the thing is useful, so I think it's more likely that they get ongoing attention/edits than that they're done and settled.

Reply:
I agree - I was considering this when putting the demos together, but since the only templating engine I'm super familiar with is React (which is very overkill for this use case, in my opinion) I didn't think the time cost of familiarizing myself with something like Pug + setting that up + re-implementing everything with templates would be worth it. I'll add an issue for this and revisit it if the pages get more numerous/complex.